Serializer/Normalizer/AbstractExceptionNormalizer.php

```diff
@@ -44,14 +44,14 @@ public function __construct(ExceptionValueMap $messagesMap, $debug)
     /**
      * Extracts the exception message.
      *
-     * @param \Exception $exception
+     * @param \Throwable $exception
      * @param int|null   $statusCode
      *
      * @return string
      */
-    protected function getExceptionMessage(\Exception $exception, $statusCode = null)
+    protected function getExceptionMessage(\Throwable $exception, $statusCode = null)
```

Comment:
You change a definition of protected method, if someone overload this method, the code will be uncompatible.

Reply:
Thank you for the heads up. In fact, here this change is safe as the `AbstractExceptionNormalizer` class is marked as `@internal` which means that we do not need to aim for BC here.